src/Sulu/Bundle/ContentBundle/Repository/NodeRepository.php

```diff
@@ -753,7 +752,7 @@ public function copyLocale($uuid, $userId, $webspaceKey, $srcLocale, $destLocale
         try {
             // call mapper function
             $structure = $this->getMapper()->copyLanguage($uuid, $userId, $webspaceKey, $srcLocale, $destLocales);
-        } catch (PHPCRException $ex) {
+        } catch (DocumentManagerException $ex) {
```

Comment:
`Doctrine\ODM\PHPCR\PHPCRException` does not exist as doctrine odm seems not be required by something. Maybe changed when document manager was added.